src/views/components/gridView.jsx

```diff
@@ -32,7 +32,8 @@ export const GridCenterView = ({
         hideSidePane && !isChatOpen ? "w-full" : "w-full md:w-4/5"
       }`}
     >
-      {peers && peers.length > 0 ? (
+      {peers &&
+      peers.filter(peer => peer.audioTrack || peer.videoTrack).length > 1 ? (
         <VideoList
```

Comment:
Shouldn't this be greater than 0?

Reply:
We were checking in the list of all peers including local peer, this has been changed now. Done